src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/DecrementInstruction.cs

```diff
@@ -29,13 +31,13 @@ public override int Run(InterpretedFrame frame)
                 }
                 else
                 {
-                    frame.Push(ScriptingRuntimeHelpers.Int32ToObject(unchecked((int)obj - 1)));
```

Comment:
There's talk elsewhere of it being a goal to have checked Debug builds and unchecked Release, which would require moving towards being explicit rather than away. As such the explicit uses here should probably stay.